CONTRIBUTING.md

```diff
@@ -29,7 +29,6 @@ If you would like to hack on MUI Toolpad or want to run the latest version, you
 
    ```sh
    yarn install
```

Comment:
there's also some text here about "building in watch mode", we might want to move that text to another step too?